### PR TITLE
Create 2017-02-15-tensorflow-framework.markdown

### PR DESCRIPTION
**Pull request by pwighton**
Brainhack Boston 2017 project proposal: Tensorflow for neuroimaging

### CODE REVIEW

File: _posts/2017-02-15-tensorflow-framework.markdown [hunk 1]
```diff
@@ -0,0 +1,12 @@
+ ---
+ layout: post
+ title: tensorflow framework for neuroimaging
+ date: YYYY-MM-DD HH:MM
```
Comment (satra) [inline _posts/2017-02-15-tensorflow-framework.markdown]: replace this with current date and time

File: _posts/2017-02-15-tensorflow-framework.markdown [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ comments: true
+ external-url:
+ categories: tensorflow
+ ---
+
+We propose to develop a framework for using tensorflow in a neuroimaging setting including:
+  - Converting neuroimaging data (`.nii.gz` format) into a format uasable by Tensorflow
+  - Developing an example 3D convolutional neural network to label voxels
```
Comment (satra) [inline _posts/2017-02-15-tensorflow-framework.markdown:12]: I would suggest adding your name as the person interested in this.

`Interested parties: Paul Wighton`

**Review by satra — COMMENTED** [review]
a few suggestions

**Comment by satra** [comment]
@pwighton - sounds cool - we have a similar ongoing project on labeling meningiomas. perhaps that can combined with this effort.

**Comment by pwighton** [comment]
Thanks @satra! I updated the project.  Looking forward to it!

**Pull request merged**